css-canvas-medical/styles.css

```diff
@@ -0,0 +1,239 @@
+.container {
+  height: 120rem;
+}
+
+.row {
+  display: flex;
+  height: 65px;
+  border-bottom: 1px solid rgb(163, 163, 163);
+}
+
+h1 {
+  padding-left: 105px;
+  font-weight: 300;
+  font-size: 45px;
+  margin-right: 370px;
+}
+
+.canvas {
+  display: flex;
+  align-items: center;
+  color: rgb(9, 9, 97);
+}
+
+.directory {
+  display: flex;
+  align-items: center;
+}
+
+button {
+  border: none;
+  background-color: white;
+  margin-left: 40px;
+  color: grey;
+  font-family: sans-serif, serif;
+}
+
+.contact-us {
+  border: 1px solid rgb(8, 8, 73);
+  border-radius: 6%;
+  background-color: rgb(8, 8, 73);
+  color: white;
+  padding: 10px;
+}
+
+.row-image {
+  margin-top: 65px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 50px;
+}
+
+.image {
+  width: 1050px;
+  height: 500px;
+  position: relative;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+  filter: brightness(0.65);
+}
+
+.image-text > h1 {
+  font-size: 35px;
+  margin-right: 100px;
+  padding-left: 47px;
+}
+
+.image-text > button {
+  color: white;
+  background-color: rgb(8, 8, 73);
+  padding: 15px 40px;
+  border-radius: 4%;
+  font-size: 15px;
+  margin-top: 25px;
+  left: 10px;
+}
+
+.image-text {
+  position: absolute;
+  width: 700px;
+  top: 45px;
+  left: 0;
+  color: white;
```

Comment:
you dont need absolute positioning here, use margin and padding to move it around in its container